sys/include/fs/xipfs_fs.h

```diff
@@ -138,6 +138,18 @@ int xipfs_extended_driver_new_file(const char *full_path, uint32_t size, uint32_
  */
 int xipfs_extended_driver_execv(const char *full_path, char *const argv[]);
 
+/**
+ * @brief Executes safely an executable file with arguments.
```

Comment:
I think this is not a good description. What does "safely execution" mean, how does the behavior change?

Reply:
That's much better. :+1: